releases/v0.13.0.md

```diff
@@ -0,0 +1,87 @@
+# Release Notes for v0.12.0
+
+Release Date: March 24th, 2025
+
+This release is based on [3.15.0](https://github.com/kata-containers/kata-containers/releases/tag/3.13.0) of Kata Containers
+and [v0.11.0](https://github.com/confidential-containers/enclave-cc/releases/tag/v0.11.0) of enclave-cc.
+
+Please see the [quickstart guide](https://confidentialcontainers.org/docs/getting-started/) for details on how to try out Confidential
+Containers.
+
+Please refer to our [Acronyms](https://github.com/confidential-containers/documentation/wiki/Acronyms)
+and [Glossary](https://github.com/confidential-containers/documentation/wiki/Glossary) pages for
+definitions of the acronyms used in this document.
+
+## What's new
+
+* AMD SEV-SNP deployments are compatible with upstream host kernels.
+* Trustee uses ECDH keys by default for KBS protocol.
+* Images with many layers can be pulled without issues.
+* Images with whiteouts are pulled correctly.
```

Comment:
Can a new section be added here, called `Bug Fix`? These two items can be added as
```
1. Fixed the issue where image pull failed due to exceeding the 4KB length limit on mount parameters caused by too many layers.
2. Fixed the issue of unexpected appearance of image whiteout files due to guest's tmpfs not supporting xattr.
```

Reply:
I moved these points to a bug fixes section but kept the wording fairly high-level.

Reply:
Ok, I thought "without issues" is a bit absolute at first and bugfix could describe the problem more accurately and could help track related issues in the future. The High-level description is also acceptable. If it is High-level, we don’t need a new section and can just keep it as it is.

Reply:
I approved the Pr. Fell free to choose the way you like

Reply:
Ok, I added the more specific stuff to the bug fixes section.